src/themes/gamedao/index.js

```diff
@@ -2,7 +2,7 @@ import { createTheme } from '@mui/material/styles'
 
 const globalTheme = {
 	shape: {
-		borderRadius: 1,
+		borderRadius: '8px',
```

Comment:
shouldnt this be derived from your shapes props below?

Reply:
I think this is not used anywhere anymore